apps/prairielearn/elements/pl-drawing/pl-drawing.mustache

```diff
@@ -10,7 +10,7 @@
 </span>
 {{/parse_error}}
 {{^parse_error}}
-<div id="pl-drawing-{{uuid}}" class="pl-drawing-container pl-requires-svg-mathjax">
+<div id="pl-drawing-{{uuid}}" class="pl-drawing-container pl-requires-svg-mathjax" aria-label="drawing-container">
```

Comment:
As discussed on Zoom, it's probably not advisable to label non-interactive elements. [MDN's docs for `aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) specifically call out that it's for interactive elements:

> **Note:** `aria-label` is intended for interactive elements or for elements made interactive via other ARIA declarations when there's no visible text in the DOM to serve as a label.

> **Note:** While aria-label can be used on any element that can have an accessible name, in practice however, it is supported only on interactive elements, [widgets](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#2._widget_roles), [landmarks](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#3._landmark_roles), images, and iframes.

Reply:
Moved the label to the canvas element in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).

Reply:
Continuing discussion here: https://github.com/PrairieLearn/PrairieLearn/pull/10514#discussion_r1737505396. I'll resolve this thread.